cli/crates/cli/tests/defer/main.rs

```diff
@@ -111,8 +111,8 @@ async fn defer_multipart_test() {
             ]
           }
         },
-        "path": [],
-        "hasNext": false
```

Comment:
The CLI workspace had the `preserve_order` feature enabled on serde_json, and the engine workspace didn't. Had to standardize one way.